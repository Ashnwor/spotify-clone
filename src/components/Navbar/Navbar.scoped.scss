.navbar {
  bottom: 89px;
  display: flex;
  width: 14.5em;
  gap: 16px;
  flex-direction: column;
  background-color: #040404;

  @media (max-width: 768px) {
    display: none;
  }
}

.logo {
  width: 131px;
  height: 40px;
  margin-top: 24px;
  margin-left: 24px;
}
