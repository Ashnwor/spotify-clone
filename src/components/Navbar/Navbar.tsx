import styles from './Navbar.module.scss';
import Nav from '../Nav/Nav';
import NavMenu from '../NavMenu/NavMenu';
// import NavList from '../NavList/NavList';
import { Logo } from '../icons';

const Navbar: React.FC = () => {
  return (
    <div className={styles.navbar}>
      <Logo className={styles.logo} />
      <Nav />
      <div style={{ display: 'flex', flexDirection: 'column', gap: 24, marginTop: 24 }}>
        <NavMenu className={styles['nav-menu']} />
        {/* <NavList /> */}
      </div>
    </div>
  );
};

export default Navbar;
