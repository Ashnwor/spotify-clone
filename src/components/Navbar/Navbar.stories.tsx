import React from 'react';
import Navbar from './Navbar';

export default {
  title: 'Navbar',
};

export const navbar = () => {
  return (
    <div style={{ height: 1440, width: 1024 }}>
      <Navbar />
    </div>
  );
};
