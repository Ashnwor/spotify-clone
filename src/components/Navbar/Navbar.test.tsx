import React from 'react';
import { shallow } from 'enzyme';
import Navbar from './Navbar';

it('runs correctly', () => {
  const wrapper = shallow(<Navbar />);
  expect(wrapper).toMatchSnapshot();
});
