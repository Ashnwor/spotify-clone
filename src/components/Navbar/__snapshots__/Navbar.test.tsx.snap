// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`runs correctly 1`] = `
<div
  className="navbar"
>
  <SvgLogo
    className="logo"
  />
  <Nav />
  <div
    style={
      Object {
        "display": "flex",
        "flexDirection": "column",
        "gap": 24,
        "marginTop": 24,
      }
    }
  >
    <NavMenu
      className="nav-menu"
    />
    <NavList />
  </div>
</div>
`;
